src/tests/utils/provider-util.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { ReactElement } from 'react';
+import { render, RenderOptions } from '@testing-library/react';
+import { Provider } from '../rollbar-react';
+
+export const renderWithProviderProps = (
+  ui: ReactElement,
+  options: RenderOptions,
+  providerProps: any
+) => {
```

Comment:
These warnings for "unexpected any" are intentional. This is raised by the lint settings as a warning, not an error. (For now at least.)